validator_derive/src/quoting.rs

```diff
@@ -491,7 +491,8 @@ pub fn quote_regex_validation(
 pub fn quote_nested_validation(field_quoter: &FieldQuoter) -> proc_macro2::TokenStream {
     let field_name = &field_quoter.name;
     let validator_field = field_quoter.quote_validator_field();
-    let quoted = quote!(result = ::validator::ValidationErrors::merge(result, #field_name, #validator_field.validate()););
+    let quoted = quote!(result = ::validator::ValidationErrors::merge(
+        result, #field_name, ::validator::Validate::validate(&#validator_field)););
```

Comment:
This line wrapping is a bit strange.  How about something like this to make it more readable?

```suggestion
    let quoted = quote!(
        result = ::validator::ValidationErrors::merge(
            result, #field_name, ::validator::Validate::validate(&#validator_field)
        );
    );
```

Reply:
that's a nice change, but otherwise i'll just keep whatever `cargo fmt` does after merging

Reply:
For the record, `cargo fmt` won't do anything with it since it's a macro where it doesn't know what the expected format is.